packages/dapp/src/components/AddMilestones.jsx

```diff
@@ -69,10 +69,14 @@ export const AddMilestones = ({ invoice, due, tokenData }) => {
       {
         type: revisedProjectAgreementType,
         src: revisedProjectAgreementSrc,
+        createdAt: Date.now().toString(),
```

Comment:
I believe this timestamp could be manipulated on the client side if someone wanted to. May not be a concern since I cannot see a practical malicious reason to do that but something to possibly consider.